README.md

```diff
@@ -55,23 +55,24 @@ To run the tests using `pytest`, you need to have the cloned mne-python reposito
 with a local pip-install instead of the mne-python package from pypi. Update your
 installation as follows:
 
-    $ git clone https://github.com/mne-tools/mne-python
+    $ git clone https://github.com/mne-tools/mne-python --depth 1
     $ cd mne-python
     $ pip uninstall mne  # uninstall pypi mne
     $ pip install -e .  # use the cloned repo for a local install of mne
+    $ pip install flake8 pytest nose coverage coveralls
```

Comment:
say that is necessary to run the tests but not for everyone.

Reply:
@sappelhoff did you address this?